apeer_dev_kit/_core.py

```diff
@@ -5,18 +5,29 @@
 from ._utility import copyfile
 
 class _core:
+    WFE_INPUT_ENV_VARIABLE = 'WFE_INPUT_JSON'
+    WFE_INPUT_FILE_NAME = '/params/WFE_input_params.json'
+
     def __init__(self):
         log.basicConfig(format='%(asctime)s [ADK:%(levelname)s] %(message)s', level=log.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+        log.info('Initializing')
+        self._outputs = {}
+        self._wfe_output_params_file = ''
+        self._input_json = self._read_inputs()
+        log.info('Found module\'s inputs to be {}'.format(self._input_json))
+
+    def _read_inputs(self):
+        ''' Read inputs either from file or from environment variable'''
         try:
-            log.info('Initializing')
-            self._outputs = {}
-            self._wfe_output_params_file = ''
-            self._input_json = json.loads(os.environ['WFE_INPUT_JSON'])
-            log.info('Found module\'s inputs to be {}'.format(self._input_json))
-        except KeyError:
-            message = 'Environment variable WFE_INPUT_JSON not found'
+            if self.WFE_INPUT_ENV_VARIABLE in os.environ:
+                return json.loads(os.environ[self.WFE_INPUT_ENV_VARIABLE])
+            else:
+                with open(self.WFE_INPUT_FILE_NAME, 'r') as input_file:
+                    return json.load(input_file)
+        except IOError:
```

Comment:
@evhen14 Hi Yevgen, why remove the KeyError? It can still happen. You can catch two errors.